packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -99,23 +105,22 @@ describe(`Test plugin manifest options`, () => {
     const icon = `pretend/this/exists.png`
     const size = 48
 
+    let pluginSpecificOptions = {
```

Comment:
This can be a const right?
```suggestion
    const pluginSpecificOptions = {
```